.github/workflows/translate.yml

```diff
@@ -29,13 +26,18 @@ jobs:
           submodules: recursive
           ref: ${{ github.ref }}
 
-      - name: l10n-push
+      - name: l10n-push-sdk
         if: github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'
         # we need to use a different docker image for those two, this appears to be a bit tricky with github actions out of the box
-        run:  bash .github/workflows/run-in-docker.sh owncloudci/transifex:latest "cd ownCloud/Resources/en.lproj && tx push -s --skip"
+        run:  bash .github/workflows/run-in-docker.sh owncloudci/transifex:latest "cd ownCloudSDK/Resources/en.lproj && tx push -s --skip"
```

Comment:
Hmm I just thought about the invocation of tx .
And I guess we should probably do it in the root dir where https://github.com/owncloud/ios-sdk/tree/master/.tx is located.
I assume that then just a single call will be needed.
This might also explain why in the ios repo the threshold was not used?

Reply:
Yes the push https://github.com/owncloud/ios-app/actions/runs/10105006407/job/27944867908 failed.